cwrapper/CArachneWrapperCTest.c

```diff
@@ -175,13 +129,18 @@ main(int argc, char** argv) {
     mainArgs.argc = argc;
     mainArgs.argv = argv;
 
-    retval = startCoreArbiter();
-    if (retval == -1) {
-        fprintf(stderr, "Faild to start CoreArbiter server! \n");
-        exit(1);
+    system("../CoreArbiter/bin/coreArbiterServer > /dev/null 2>&1 &");
+
+    /* Wait at most 10 sec for Arachne to initialize */
+    for (int i = 0; i < 1000; ++i) {
+        retval = cArachneInit(&argc, (const char**)argv);
+        if (retval == 0) {
+            break;
+        } else {
+            usleep(10000);  // Wait 10ms before retry
+        }
     }
 
-    cArachneInit(&argc, (const char**)argv);
     retval = cArachneCreateThread(&threadId, mainTestLoop, (void*)&mainArgs);
```

Comment:
If Arachne failed to initialize, I think we should detect this before trying to create a new thread and immediately exit with an error message. Otherwise, the test will print the confusing floating point exception again.